crates/epoch-snark/tests/e2e.rs

```diff
@@ -34,4 +35,25 @@ fn prover_verifier_groth16() {
     // Verifier checks the proof
     let res = verify(&params.epochs.vk, &first_epoch, &last_epoch, &proof);
     assert!(res.is_ok());
+
+    // Serialize the proof / vk
+    let mut serialized_vk = vec![];
```

Comment:
This is added because the serialized output is then used in the verify-only FFI test